### PR TITLE
Do proxy setting parsing after config.py is imported and parsed

### PR DESCRIPTION
**Pull request by dmuntean**
Proxy parsing should happen after import of config.py, otherwise the only way to set PROXY_URL is via environment.

### CODE REVIEW

**Comment by skoczen** [comment]
@dmuntean Yep, you are 100% correct.  Thanks for catching this, and submitting a PR to fix it!

Will get this merged in on my next will sprint.  

Would you mind adding yourself to AUTHORs on this PR, with the attribution you'd prefer?

Thanks for making will better!

**Comment by dmuntean** [comment]
It was actually me who added the proxy support in the first place, so my details are already in AUTHORS.

**Comment by skoczen** [comment]
My goodness.  Yep, my mistake completely - going too fast in a quick review!

Let me try again - thanks so much for keeping your improvements working and catching things I missed!

**Comment by skoczen** [comment]
This is now released as [v0.9](http://skoczen.github.io/will/improve/#09-may-29-2015)!  Thanks again!
